frontend/src/components/successMessage/successMessage.module.css

```diff
@@ -1,11 +1,5 @@
-.successMessageContainer {
-  display: flex;
-  height: 100vh;
-}
-
```

Comment:
This styling looked better on desktop, but it looks worse on mobile so I think we're better of reverting it.